.eslintrc.js

```diff
@@ -0,0 +1,28 @@
+module.exports = {
+	'extends': [
+		'./eslint/config.js',
+		'plugin:jest/recommended',
+	],
+	'plugins': [
+		'wordpress',
+	],
+	env: {
+		'jest/globals': true,
+	},
+	globals: {
+		wpApiSettings: true,
+		eejsdata: true
+	},
+	plugins: [
+		'jest',
+	],
+	overrides: [
+		{
+			files: [ 'test/e2e/**/*.js' ],
```

Comment:
I left this in ... eventually I plan on adding e2e (end to end) tests for our javascript work.  The folder structure may change then and I'll modify this if/when that happens.